python/core-sdk/alation_ai_agent_sdk/api.py

```diff
@@ -477,6 +477,48 @@ def get_context_from_catalog(self, query: str, signature: Optional[Dict[str, Any
                 help_links=["https://developer.alation.com/"],
             )
 
+    def get_bulk_objects_from_catalog(self, signature: Dict[str, Any]):
+        """
+        Retrieve bulk objects from the Alation catalog based on signature specifications.
+        Uses the context API in bulk mode without requiring a natural language question.
+        """
+        if not signature:
+            raise ValueError("Signature cannot be empty for bulk retrieval")
+
+        self._with_valid_token()
+
+        headers = {
+            "Token": self.access_token,
+            "Accept": "application/json",
+        }
+
+        params = {
+            "mode": "bulk",
+            "signature": json.dumps(signature, separators=(",", ":"))
+        }
+
+        encoded_params = urllib.parse.urlencode(params, quote_via=urllib.parse.quote)
+        url = f"{self.base_url}/integration/v2/context/?{encoded_params}"
+
+        try:
+            response = requests.get(url, headers=headers, timeout=60)
+            response.raise_for_status()
+
+        except requests.RequestException as e:
+            self._handle_request_error(e, "bulk catalog retrieval")
+
+        try:
+            return response.json()
+        except ValueError:
+            raise AlationAPIError(
+                message="Invalid JSON in bulk catalog response",
+                status_code=response.status_code,
+                response_body=response.text,
+                reason="Malformed Response",
+                resolution_hint="The server returned a non-JSON response. Contact support if this persists.",
+                help_links=["https://developer.alation.com/"],
```

Comment:
Should we link the page to the context api? https://developer.alation.com/dev/reference/getaggregatedcontext